src/schema/2.0/schema.json

```diff
@@ -8,23 +8,39 @@
       "properties": {
         "type": {
           "type": "string",
-          "enum": ["DNS-TXT"]
+          "enum": ["DNS-TXT", "DID", "DNS-DID"]
         },
         "location": {
           "type": "string",
           "description": "Url of the website referencing to document store"
+        },
+        "key": {
+          "type": "string",
+          "description": "Public key associated"
         }
       },
-      "required": ["type", "location"],
-      "additionalProperties": false
+      "required": ["type"]
```

Comment:
there is a regression here, location is no longer mandatory when type is DNS-TXT and you wont be aware before using verify. 

We can be stricter and use oneOf schema way ? (that should set the type correctly) or do the verification manually? (type will be incorrect)

Reply:
done